*** SpringRooSecurity/pom.xml	2011-08-17 15:08:05.755282000 +0700
--- SecAdvDemo/pom.xml	2011-08-17 11:06:12.255282001 +0700
***************
*** 156,162 ****
      <dependency>
              <groupId>org.hsqldb</groupId>
              <artifactId>hsqldb</artifactId>
!             <version>1.8.0.10</version>
              <classifier/>
          </dependency>
          <dependency>
--- 156,162 ----
      <dependency>
              <groupId>org.hsqldb</groupId>
              <artifactId>hsqldb</artifactId>
!             <version>2.2.4</version>
              <classifier/>
          </dependency>
          <dependency>
***************
*** 486,492 ****
                  <artifactId>maven-surefire-plugin</artifactId>
                  <version>2.8</version>
                  <configuration>
!                     <printSummary>false</printSummary>
                      <redirectTestOutputToFile>true</redirectTestOutputToFile>
                      <excludes>
                          <exclude>**/*_Roo_*</exclude>
--- 486,492 ----
                  <artifactId>maven-surefire-plugin</artifactId>
                  <version>2.8</version>
                  <configuration>
!                     <printSummary>true</printSummary>
                      <redirectTestOutputToFile>true</redirectTestOutputToFile>
                      <excludes>
                          <exclude>**/*_Roo_*</exclude>
